html2docx.py

```diff
@@ -1,32 +1,34 @@
 import io
 import re
 from html.parser import HTMLParser
+from typing import List, Tuple
 
-import docx
+from docx import Document
+from docx.text.paragraph import Paragraph
 
 WHITESPACE_RE = re.compile(r"\s+")
 
 
 class HTML2Docx(HTMLParser):
     header_re = re.compile(r"^h[1-6]$")
 
-    def __init__(self, title):
+    def __init__(self, title: str):
         super().__init__()
-        self.doc = docx.Document()
+        self.doc = Document()
         self.doc.core_properties.title = title
 
-        self.p = None
-        self.runs = []
-        self.list_style = []
+        self.p: Paragraph = None
+        self.runs: List[Run] = []
+        self.list_style: List[str] = []
         self.bold = 0
```

Comment:
Why not annotate these members?

Reply:
mypy can infer their type. They must be an int. mypy can't do the same for an empty list that should contain only a specific type.